draft-ietf-tls-tls13.md

```diff
@@ -1375,12 +1368,10 @@ For example:
        } V2;
 
        struct {
-           select (VariantTag) {
-               case apple:
-                 V1;   /* VariantBody, tag = apple */
-               case orange:
-               case banana:
-                 V2;   /* VariantBody, tag = orange or banana */
+           VariantTag type;
+           select (VariantRecord.type) {
+               case apple:  V1;  /* VariantBody, tag = apple */
```

Comment:
I'd remove it entirely as I don't see any of these VariantBody and tag things used anywhere in the spec.